js/lib/bands.js

```diff
@@ -139,6 +139,20 @@ function bandPlot(bandDivId, bandPathTextBoxId, dataFilePaths, dosFile, showFerm
                 a.click();
             }
         }
+
+        var theDownloadJsonButton = document.getElementById(bandDivId + "bt-downloadJson");
+        theDownloadJsonButton.onclick = function () {
+            if ($.isEmptyObject(dosFile)) {
+                var a = document.createElement('a');
+                if (dataFilePaths.length) {
```

Comment:
I don't understand this for loop, it will go through every band and download or it will download all as a archived file?